package.json

```diff
@@ -149,7 +149,6 @@
   "trackingCode": "UA-49225444-1",
   "greenkeeper": {
     "ignore": [
-      "diff"
```

Comment:
I thought about it, but decided to leave it so that I remember where the ignores go for next time 😝